From cd8eea954de1e5bc39a04d23ddfd1088356da9c6 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Mon, 3 Feb 2025 12:42:50 +0100
Subject: [PATCH] media: dcmipp: pixelproc: reuse default height/width from
 common

Avoid duplicated define of default HEIGHT and WIDTH
which are already defined within the dcmipp-common.h file.

Change-Id: I853c448ad26044d4f6b1cd89afad8eeadcc8ad6f
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/436012
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Domain-Review: Hugues FRUCHET <hugues.fruchet@st.com>
Tested-by: Hugues FRUCHET <hugues.fruchet@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 .../media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelproc.c    | 3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelproc.c
+++ b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelproc.c
@@ -18,9 +18,6 @@
 
 #include "dcmipp-common.h"
 
-#define DCMIPP_FMT_WIDTH_DEFAULT  640
-#define DCMIPP_FMT_HEIGHT_DEFAULT 480
-
 #define DCMIPP_P1FCTCR	0x900
 #define DCMIPP_P2FCTCR	0xD00
 #define DCMIPP_PxFCTCR(id) (((id) == 1) ? DCMIPP_P1FCTCR :\
